.travis.yml

```diff
@@ -14,7 +14,7 @@ addons:
 sudo: false
 
 install:
- - npm install --fallback-to-build=false
+ - yarn install --fallback-to-build=false
```

Comment:
This is also failing because of our custom `mapnik` build??

Reply:
Yes, it is. To be honest, I don't know why that combination of npm version and our custom module fails.